content/_data/changelogs/weekly.yml

```diff
@@ -6948,6 +6948,97 @@
       authors:
         - jvz
 
+- version: 2.229
+  date: 2020-03-29
+  changes:
+  - type: major bug
+    category: major bug
+    pull: 4602
+    issue: 61688
+    authors:
+    - daniel-beck
+    message: |-
+      Actually load a previously saved global build discarder configuration after restart.
+  - type: major bug
+    category: regression
+    pull: 4607
+    issue: 61692
+    authors:
+    - daniel-beck
+    message: |-
+      Fix proxy form validation when a password is set (regression in 2.205).
+  - type: rfe
+    category: rfe
+    pull: 4554
+    authors:
+    - NextTurn
+    message: |-
+      Update .NET version checks to be more correct for modern .NET versions.
+  - type: rfe
+    category: rfe
+    pull: 4569
+    issue: 61455
+    authors:
+    - amuniz
+    - timja
+    message: |-
+      <code>About Jenkins</code> management link is now accessible to users with <code>Overall/Manage</code> or <code>Overall/SystemRead</code> (as well as the usual <code>Overal/Administer</code>).
+  - type: bug
+    category: bug
+    pull: 4608
+    authors:
+    - daniel-beck
+    message: |-
+      Robustness: Don't throw a <code>NullPointerException</code> when trying to convert <code>null</code> to <code>Secret</code>.
+  - type: bug
+    category: bug
+    pull: 4601
+    issue: 61409
+    authors:
+    - jeffret-b
+    - jglick
+    - Vlatombe
+    references:
+      - pull: 4601
+      - pull: 4596
+      - issue: 61409
+      - url: https://github.com/jenkinsci/remoting/releases/tag/remoting-4.3
+        title: Remoting 4.3 changelog
+      - url: /blog/2020/02/02/web-socket/
+        title: WebSockets blog post
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/222/README.adoc
+        title: JEP-222
+    message: |-
+      Upgrade to Remoting 4.3 to fix an issue with large payloads over WebSockets.
+      Requires a matching <code>agent.jar</code> with remoting 4.3 or later.
+  - type: rfe
+    category: developer
+    pull: 3941
+    issue: 56643
+    authors:
+    - jsoref
+    message: |-
+      Create symlinks atomically to prevent lastSuccessfulBuild link from being randomly missing.
+  - type: rfe
+    category: internal
+    pull: 4603
+    authors:
+    - jglick
+    message: |-
+      Developer: <code>Secret</code> and <code>ConfidentialKey</code> implementations can now be used from unit tests without <code>JenkinsRule</code>.
+  - type: rfe
+    category: internal
+    pull: 4605
+    authors:
+    - jglick
+    message: |-
+      Internal: Do not print random binary data during WebSocketAgentsTest.
```

Comment:
Ignore? It is not really helpful to Jenkins users IMHO, just a test fix. No changes in the production code

Reply:
Thanks for the feedback!  Fixed in b290df20f832f96fbbd8ec390408c3ad9488d20f, will be part of an upcoming pull request.